clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -769,6 +773,7 @@ private static class TopicPartitionState {
         private Long logStartOffset; // the log start offset
         private Long lastStableOffset;
         private boolean paused;  // whether this partition has been paused by the user
+        private boolean consumable;
```

Comment:
How about calling this `pendingRevocation` or something like that? That might make the usage clearer.

Reply:
I like the name: `pendingRevocation`, too.